captum/attr/_core/feature_ablation.py

```diff
@@ -0,0 +1,329 @@
+                            should contain the same number of tensors as inputs,
+                            and features within each input tensor are ablated
+                            independetly (not across tensors). Each tensor should
+                            be the same size as the corresponding input or
+                            broadcastable to match the input tensor. Each tensor
+                            should contain integers in the range 0 to num_features
+                            - 1, and indices corresponding to the same feature should
+                            have the same value.
+                            If None, then a feature mask is constructed which assigns
+                            each scalar within a tensor as a separate feature, which
+                            is ablated independently.
+                            Default: None
+                baselines (scalar, tensor, tuple of scalars or tensors, optional):
+                            Baselines define reference value which replaces each feature
+                            when ablated. In order to assign attribution scores DeepLift
+                            computes the differences between the inputs and references
+                            and corresponding outputs.
+                            Baselines can be provided either as :
+
+                            - a single tensor, if inputs is a single tensor, with
+                                exactly the same dimensions as inputs (first dimension
+                                can be 1, which applies the same baseline to all
+                                input examples).
+
+                            - a tuple of tensors, if inputs is a tuple of tensors,
+                                with matching dimensions to inputs (first dimension
+                                can be 1, which applies the same baseline to all
+                                input examples).
+
+                            - a single scalar, if inputs is a single tensor, which will
+                                be broadcasted for each input value in input tensor.
+
+                            - a tuple of scalars, if inputs is a tuple of tensors, with
+                                exactly the same number of elements as inputs tuple.
+                                Each scalar element in baselines' tuple is broadcasted
+                                for each input tensor at the same index in inputs
+                                tuple.
+                            Default: zero scalar for each input tensor
+                internal_batch_size (int, optional): Divides total #features *
+                            #examples data points into chunks of size
+                            internal_batch_size, which are computed sequentially.
+                            internal_batch_size should be a multiple of #examples.
```

Comment:
Is this condition being enforced?

Reply:
No, it's just implicitly being rounded down to the nearest multiple of #examples. But that's a good point, will add a line of documentation for this, thanks!